src/core_modules/capture-core/components/Pages/ViewEvent/ViewEventComponent/viewEvent.actions.js

```diff
@@ -22,7 +22,7 @@ export const actionTypes = {
 
 export const viewEventFromUrl = (data: Object) =>
     actionCreator(actionTypes.VIEW_EVENT_FROM_URL)({
-        eventId: data.nextProps.viewEventId,
+        eventId: data.nextProps.viewEventId || data.nextProps.eventId,
```

Comment:
Hey, 
I don't think this change is needed anymore as the URL will not contain the parameter `eventId`. 
Thanks!